dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/DatadogCore.kt

```diff
@@ -128,9 +128,10 @@ internal class DatadogCore(
     /** @inheritDoc */
     override fun registerFeature(feature: Feature) {
         val sdkFeature = SdkFeature(
-            coreFeature,
-            feature,
-            internalLogger
+            featureSdkCore = this,
```

Comment:
Alternatively we can just add new method to `InternalLogger`. But not a big deal.